### ISSUE TITLE
Increasing WER & Validation Loss During Whisper Fine-Tuning

### ISSUE THREAD

**Issue by monk1337** [huggingface/community-events#197]
Hello everyone,

I've recently created a dataset using speech-to-text APIs on custom documents. The dataset consists of 1,000 samples, with 700 designated for training and 300 for testing. In total, this equates to about 4 hours of audio, where each clip is approximately 30 seconds long.

I'm attempting to fine-tune the Whisper small model with the help of HuggingFace's script, following the tutorial they've provided [Fine-Tune Whisper For Multilingual ASR with 🤗 Transformers](https://huggingface.co/blog/fine-tune-whisper).

Before diving into the fine-tuning, I evaluated the WER on OpenAI's pre-trained model, which stood at WER = 23.078%.

However, as my fine-tuning progresses, I'm observing some unexpected behavior:

<img width="561" alt="Screenshot 2023-11-01 at 2 11 14 AM" src="https://github.com/huggingface/community-events/assets/17107749/3c744da5-db73-4448-b0e3-b91794b8e2d2">

As visible, the Validation Loss and WER are both on the rise during the fine-tuning phase. I'm at a bit of a loss here. Why might this be happening? Any insights or recommendations would be greatly appreciated.

Thank you in advance!
@Vaibhavs10 @sanchit-gandhi

**Comment by abcnorio** [comment]
May I ask where to set the dropout and activation_dropout parameters exactly? I could find them in a whisper final config as an output of whisper finetuning but how to apply that during training? Which training argument points to that? 

Thanks!
